src/tests/stdlib/strcat.c

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file
+ *
+ * @date Mar 31, 2020
+ * @author: Prajawal Kumar Pandey
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("test suite for strcat()");
+
+TEST_CASE("concatenate full string with strcat()") {
+    char src[] = "string";
+    char dst[] = "char ";
+
+    strcat(dst, src);
```

Comment:
Is there buffer overflow?
will a final string became "char string"?

sizeof(dst) (sizeof("char ")) < sizeof(""char string")